pkg/tournament/classic_division.go

```diff
@@ -406,11 +407,41 @@ func (t *ClassicDivision) SubmitResult(round int,
 
 	pk1, err := t.getPairingKey(p1, round)
 	if err != nil {
-		return nil, err
+		found := false
```

Comment:
Shouldn't we check that we are actually in IRL mode at some point in this if block?

Reply:
yeah, you're probably right.